code/datums/gamemode/vampire_gamemode.dm

```diff
@@ -1,38 +1,77 @@
 /datum/gamemode/vampire
-	minimum_player_count = 10
+	minimum_player_count = 0 // OVERRIDE
 	name = "Vampire"
 	factions_allowed = list(/datum/faction/vampire)
 
+	roles_allowed = 1 // Per factions
+
+	var/number_of_vamps
+
+/datum/gamemode/vampire/CreateFactions()
```

Comment:
Same as above. There has to be a better way.

Reply:
I based it on the idea of 1 vampire = 1 faction, which is quite unique to the mode and leave more room for expansion than letting all vamps be orphaned roles.